gallery/src/styles/NxFontSize/NxFontSizePage.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const nxFontSizeHtmlExampleCode = require('!!raw-loader!./NxFontSizeHtmlExample.html').default;
+const nxFontSizeScssExampleCode = require('!!raw-loader!./NxFontSizeScssExample.scss').default;
+
+const NxFontSizePage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">nx-font-size()</code> is a mixin that can be applied to the
+        {' '}<code className="nx-code">&lt;body&gt;</code> tag of your project to adjust the default RSC font size to
+        better match your project. The default font size in RSC is 16px.
+      </p>
+      <p className="nx-p">
+        The mixin takes a single variable which is the desired font-size with unit. For example:
+        {' '}<code className="nx-code">@include nx-font-size(14px);</code> or
+        {' '}<code className="nx-code">@include nx-font-size(1.1em);</code>.
+      </p>
+      <p className="nx-p">
+        The mixin adjusts the default text size as well as several user agent set form element font sizes.
```

Comment:
"user agent set form element font sizes" is pretty clunky; consider rewording.

Reply:
> "user agent set form element font sizes" is pretty clunky; consider rewording.

Yeah, that wasn't great, how's this: https://github.com/sonatype/sonatype-react-shared-components/pull/92/commits/b997b6b56d0d6584116ae9f5e611320992c82bd8